Sources/SwiftSyntaxBuilder/TupleExprElementConvenienceInitializers.swift

```diff
@@ -17,6 +17,6 @@ public extension TupleExprElement {
   /// The presence of the colon will be inferred based on the presence of the label.
   init(label: String? = nil, expression: ExpressibleAsExprBuildable) {
     self.init(
-      label: label.map(TokenSyntax.identifier), colon: label == nil ? nil : .colon, expression: expression)
+      label: label.map { TokenSyntax.identifier($0) }, colon: label == nil ? nil : .colon, expression: expression)
```

Comment:
Isn't it equivalent to what it was?

Reply:
The old implementation doesn’t work anymore because `TokenSyntax.identifier` now takes three arguments, two of which are defaulted. But the argument default values are not used when referencing the function by its base name.